src/helpers.ts

```diff
@@ -508,38 +518,37 @@ export async function getInstantMixItems(
     userId: string,
     item: BaseItemDto
 ): Promise<BaseItemDtoQueryResult> {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    const query: any = {
-        Fields: requiredItemFields,
-        Limit: 50,
-        UserId: userId
+    if (item.Id === undefined) {
+        throw new Error('Item ID not provided');
+    }
+
+    const query: InstantMixApiRequest = {
+        fields: ['MediaSources', 'Chapters'],
+        itemId: item.Id,
+        limit: 50,
+        userId
```

Comment:
User id doesn't have to be provided as it is inferred from the access token since Jellyfin 10.9